code/code/misc/damage.cc

```diff
@@ -27,6 +27,229 @@
 #define DAMAGE_DEFINE 0
 #define DAMAGE_DEBUG 0
 
+namespace {
+
+  void deathShout(TBeing* v, TBeing* being) {
```

Comment:
nit: Can it be `nullptr`? Probably not? Then we could use references, instead of pointers - use the typesystem to describe non-nullness.